ice.c

```diff
@@ -2418,11 +2418,14 @@ void janus_ice_candidates_to_sdp(janus_ice_handle *handle, char *sdp, guint stre
 #endif
 			}
 		}
-		g_strlcat(sdp, buffer, JANUS_BUFSIZE);
+#pragma GCC diagnostic ignored "-Wformat-security"
```

Comment:
> At present, this warns about calls to "printf" and "scanf" functions where the format string is not a string literal and there are no format arguments, as in `printf (foo);`.  This may be a security hole if the format string came from untrusted input and contains `%n`.

The fix should be just

```c
janus_sdp_attribute_create("candidate", "%s", buffer);
```